### ISSUE TITLE
`CreateFile` does not generate generic access flags

### ISSUE THREAD

**Issue by colejohnson66** [microsoft/CsWin32#858]
## Actual behavior

As the title implies, referencing `CreateFile` does not generate `GENERIC_READ` and `GENERIC_WRITE` for `FILE_ACCESS_FLAGS`. They don't exist, and I'm forced to create my own:

```cs
const FILE_ACCESS_FLAGS GENERIC_WRITE = (FILE_ACCESS_FLAGS)0x4000_0000;
const FILE_ACCESS_FLAGS GENERIC_READ = (FILE_ACCESS_FLAGS)0x8000_0000;
```

## Expected behavior

Said flags are generated into the enum.

## Repro steps

1. `NativeMethods.txt` content:
```
CreateFileW
```

### Context

- CsWin32 version: `0.2.188-beta`
- Target Framework: `net7.0`

**Comment by AArnott** [comment]
Good point. [The docs](https://learn.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-createfilew#parameters) state that:

> The most commonly used values are GENERIC_READ, GENERIC_WRITE

And yet they are not defined on the `FILE_ACCESS_FLAGS` enum that the 2nd parameter is typed as. 
I _do_ see the constants defined in the metadata, but they are typed as `uint` instead of as `FILE_ACCESS_FLAGS`.

Moving to metadata for resolution.
